src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -33,6 +34,9 @@ const ModalWindow: FC<IChange> = ({ type, message, date, expDate, idd }) => {
     ) {
       if (!type) {
         dispatch(addModalTodo(input));
+        if (filtValue === "active" || filtValue === "completed") {
```

Comment:
Try to create enum for 'active', 'completed' and use it

Reply:
Fixed